media/css/mozorg/technology.scss

```diff
@@ -878,11 +807,11 @@ html[dir="rtl"] #technologies > section {
         }
 
         p {
-            @include font-size(16px);
+            @include font-size-level5;
```

Comment:
Yeah... maybe we should stick with a hard `font-size(16px)` here since 14px really is too small for body copy in Zilla. Perhaps standard body copy should be the common exception to the "use size mixins" rule because I really don't want it to ever go smaller than 16px. Or we could even add another mixin like `font-size-body` that starts at 16px and goes to 18px on desktop? Maybe that's overthinking? Really this page just needs to be redesigned to follow our new type conventions.